app/views/devise/passwords/edit.html.haml

```diff
@@ -4,20 +4,17 @@
   %span
 .entry-content.wpcf7
   = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
-    - if resource.errors.any?
-      .wpcf7-response-output
-        - resource.errors.full_messages.each do |msg|
-          = msg
+
+    != model_errors_helper(resource)
+    %br/
```

Comment:
I like the break between the errors div and the next section of the form.   How about adding that to the helper method as part of this PR?